<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1166</width>
    <height>798</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="mainview">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>811</width>
      <height>751</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>829</x>
      <y>9</y>
      <width>331</width>
      <height>781</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>301</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Функции</string>
     </property>
     <widget class="QComboBox" name="funcs">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>301</width>
       <height>201</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Ввод значений</string>
     </property>
     <widget class="QLineEdit" name="x_min_text">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>-5</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Xmin</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="z_min_text">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>-5</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Zmin</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="x_max_text">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Xmax</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="z_max_text">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>70</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Zmax</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="delta_x_text">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>37</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ΔX</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>37</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ΔZ</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="delta_z_text">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="draw_all_btn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
                                  display: inline-block;
                                  font-size: 16px;
                                  cursor: pointer;
                                  text-align: center;
                                  text-decoration: none;
                                  outline: none;
                                  color: #fff;
                                  background-color: #4CAF50;
                                  border: none;
                                  border-radius: 10px;
                                  box-shadow: 0 9px #999;
                                }
QPushButton:hover {background-color: #3e8e41}
QPushButton:pressed {
                                  background-color: #3e8e41;
                                  box-shadow: 0 5px #666;
                                  transform: translateY(4px);
                                }</string>
      </property>
      <property name="text">
       <string>Отобразить</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>301</width>
       <height>201</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Поворот</string>
     </property>
     <widget class="QLineEdit" name="delta_theta_OX_text">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>OX (градусы)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="delta_theta_OY_text">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>69</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>OY (градусы)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="delta_theta_OZ_text">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>OZ (градусы)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rotate_btn">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>150</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
                                  display: inline-block;
                                  font-size: 16px;
                                  cursor: pointer;
                                  text-align: center;
                                  text-decoration: none;
                                  outline: none;
                                  color: #fff;
                                  background-color: #4CAF50;
                                  border: none;
                                  border-radius: 10px;
                                  box-shadow: 0 9px #999;
                                }
QPushButton:hover {background-color: #3e8e41}
QPushButton:pressed {
                                  background-color: #3e8e41;
                                  box-shadow: 0 5px #666;
                                  transform: translateY(4px);
                                }</string>
      </property>
      <property name="text">
       <string>Повернуть</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="clear_all_btn">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>520</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
                              display: inline-block;
                              font-size: 16px;
                              cursor: pointer;
                              text-align: center;
                              text-decoration: none;
                              outline: none;
                              color: #fff;
                              background-color: #DC143C;
                              border: none;
                              border-radius: 10px;
                              box-shadow: 0 9px #999;
                           }
QPushButton:hover {background-color: #B22222}
QPushButton:pressed {
                              background-color: #B22222;
                              box-shadow: 0 5px #666;
                              transform: translateY(4px);
                            }</string>
     </property>
     <property name="text">
      <string>Очистить</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
